amy/fiscal/forms.py

```diff
@@ -147,35 +148,35 @@ def clean(self):
 class MembershipCreateForm(MembershipForm):
     comment = MarkdownxFormField(
         label="Comment",
-        help_text="This will be added to comments after the membership is " "created.",
+        help_text="This will be added to comments after the membership is created.",
         widget=forms.Textarea,
         required=False,
     )
 
-    helper = BootstrapHelper(
-        submit_label="Save membership and go to next screen", add_cancel_button=True
+    helper = BootstrapHelper(add_cancel_button=True)
+
+    main_organization = forms.ModelChoiceField(
+        Organization.objects.all(),
+        label="Main organisation",
```

Comment:
Good catch. I asked @maneesha recently about this, so let me paste her answer below:

> [...] our style guide dictates using British English.  For AMY, I think the front end reflects this but the back end code uses spellings like "organization" instead of "organisation."   While it would be nice to keep this consistent in the user interface, I don't think we need to prioritize this on the back end right now.

Some changes may be difficult to conduct now, e.g. changing the name of the model `Organization` -> `Organisation`. So at least I try to keep the UI-related spelling in British English.

Reply:
Oh! I'm glad I asked.